assets/app/scripts/directives/nav.js

```diff
@@ -5,7 +5,7 @@ angular.module('openshiftConsole')
     return {
       restrict: 'E',
       templateUrl: 'views/_sidebar.html',
-      link: function($scope, element, attrs) {
+      link: function($scope) {
```

Comment:
For framework functions I tend to agree, they could be left in.  However, I'd prefer they also maintain the leading `$`, as `($scope, $element, $attrs)`.  Thoughts?